data/pages/faq-grantees.mdx

```diff
@@ -94,6 +94,8 @@ That’s basically it![^fn-legal]
 Every grantee must submit a progress report every 90 days. A markdown template
 is provided when we request your first report.
 
+If your grant started in January, for example, the 1st report will be due anytime during the month of April and should cover the work done from January thru March, and so on and so forth. You have a 30-day window to work on the report and we appreciate if you keep them on schedule.
```

Comment:
I'd suggest:
You have a 30 day window to submit the report and we appreciate if you submit them on time.

Reply:
[Commit here](https://github.com/OpenSats/website/pull/336/commits/eb32a0ecc49ab42cdc0c0b7f7aa7e69315c25d57)